paasta_tools/utils.py

```diff
@@ -853,6 +854,33 @@ def get_extra_volumes(self) -> List[DockerVolume]:
         <https://mesosphere.github.io/marathon/docs/native-docker.html>`_"""
         return self.config_dict.get("extra_volumes", [])
 
+    def get_boto_mount(self):
+        try:
+            instance_config = load_service_instance_config(
+                self.service,
+                self.instance,
+                self.get_instance_type(),
+                self.cluster,
+                self.soa_dir,
+            )
+        except paasta_tools.utils.NoConfigurationForServiceError:
+            return []
+        boto_keys = instance_config.get("boto_keys")
+        if not boto_keys:
+            return []
+        service_name = self.service + "." + self.instance
+        create_service_boto_folder(service_name, boto_keys)
+        boto_mount_path = BOTO_SOURCE_DIR + service_name
```

Comment:
Instead of having both this method and `create_service_boto_folder` build the `boto_mount_path`, it'd be nicer to just have `create_service_boto_folder` return the path to avoid the possibility that we change this in one place and forget to change it in the other